projects/storefrontlib/src/cms-components/configurator/commons/config-overview-attribute/config-overview-attribute.component.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { NgSelectModule } from '@ng-select/ng-select';
+import { I18nTestingModule } from '@spartacus/core';
+import { ConfigAttributeHeaderComponent } from '../config-attribute-header/config-attribute-header.component';
+import { ConfigOverviewAttributeComponent } from './config-overview-attribute.component';
+
+describe('ConfigurationOverviewFormComponent', () => {
+  let component: ConfigOverviewAttributeComponent;
+  let fixture: ComponentFixture<ConfigOverviewAttributeComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [I18nTestingModule, ReactiveFormsModule, NgSelectModule],
+      declarations: [ConfigOverviewAttributeComponent],
+      providers: [],
+    })
+      .overrideComponent(ConfigAttributeHeaderComponent, {
+        set: {
+          changeDetection: ChangeDetectionStrategy.Default,
+        },
+      })
+      .compileComponents();
+  }));
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ConfigOverviewAttributeComponent);
+    component = fixture.componentInstance;
+  });
+
+  it('should create component', () => {
```

Comment:
Should we test that one can pass values to this component?

Reply:
Done.